answer/ponpain/HTML/03/form.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>form</title>
+</head>
+<body>
+    <form action="" method="get">
+        <dl>
+             <dt>
+                 <p>name:</p>
+             </dt>
+             <dd>
+                 <input type="text" id="mame" name="name" placeholder="Yamada Taro" required>
+             </dd>
+             <dt>
+                 <p>email:</p>
+             </dt>
+             <dd>
+                 <input type="email" id="email" name="email" placeholder="example@gmail.com">
+             </dd>
+             <dt>
+                 <p>password:</p>
+             </dt>
+             <dd>
+                 <input type="password" id="password" name="password" placeholder="p@ssword">
+             </dd>
+             <dt>
+                 <p>age:</p>
+             </dt>
+             <dd>
+                 <p><select name="age" id="age">
```

Comment:
slectタグをp tagで挟むとしたら
```suggestion
                 <p>
                     <select name="age" id="age">
```
みたいにインデントしてほしいです 👀